api/src/main/java/org/apache/cloudstack/api/command/user/project/CreateProjectCmd.java

```diff
@@ -116,7 +116,7 @@ public long getEntityOwnerId() {
             throw new InvalidParameterValueException("Account name and domain id must be specified together");
         }
 
-        if (userId != null && (accountId == null && domainId == null)) {
+        if (userId != null && (accountId == null || domainId == null)) {
             throw new InvalidParameterValueException("Account ID and Domain ID must be specified with userID");
```

Comment:
@Pearl1594 update this msg to reflect the change in cond.

Reply:
It already did, as far as I can tell, neither accountId nor domainId may be null so both must be specified.

Reply:
am i missing anything here @DaanHoogland , the cond. is OR and the msg says both (account id and domain id).

Reply:
throw an exception when either is null, would mean both may not be null. The message was already the (not yet implemented) new condition.

Reply:
hmmm , correct @DaanHoogland